src/schema/resolvers.js

```diff
@@ -95,6 +95,10 @@ function buildOracleFilters({
     filter.token = token;
   }
 
+  if(excludeSelfAddress) {
```

Comment:
make this an `else` block after `if (resultSetterQAddress) {`. otherwise you are just overwriting if there is a `resultSetterQAddress filter`.

actually we should be doing a check if its one or the other. but, can add these safety type checks later.